package.json

```diff
@@ -71,6 +71,8 @@
     "@vitest/coverage-v8": "^1.1.0",
     "@vitest/ui": "^1.1.0",
     "classnames": "^2.3.2",
+    "echarts": "^5.4.3",
```

Comment:
Interesting :)... I've not seen this library before, I'll need to check the bundle size but I'd imagine it's tree shakable

Reply:
run `npx vite-bundle-visualizer` if you want check the bundle size 👍🏻